src/style/layers.css

```diff
@@ -146,3 +121,47 @@ ngm-catalog .ngm-checkbox {
   margin-left: 4px;
   color: #212529
 }
+
+/* KML upload */
+ngm-layers-upload button {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.25px;
+  font-weight: bold;
+  color: #0B7285;
+  background-color: #F1F3F5;
+  border: 2px dashed #0B7285;
+  margin-top: 9px;
+  padding-left: 10px;
+  height: 46px;
+  width: 100%;
+}
+
+ngm-layers-upload .ngm-upload-icon {
+  pointer-events: none;
+  background-color: #0B7285;
+  margin-top: 0;
+  margin-left: 12px;
+  width: 20px;
+  height: 20px;
+}
+
+ngm-layers-upload .ngm-upload-icon {
+  pointer-events: none;
+  background-color: #0B7285;
+  margin-top: 0;
+  margin-left: 12px;
+  width: 20px;
+  height: 20px;
+}
+
+/* FIXME: Grey out action menue until further specification */
+ngm-layers-upload .ui.dropdown {
```

Comment:
it is ok to keep it like this but you can also use `ngm-disabled` instead of adding new styles

Reply:
Elisabeth is ok with grey out, so we might just leave it, anyways it will get removed soon.